corehq/apps/registration/forms.py

```diff
@@ -493,18 +496,28 @@ def __init__(self, data=None, excluded_emails=None, is_add_user=None, location=N
         super(AdminInvitesUserForm, self).__init__(data=data, **kwargs)
         domain_obj = Domain.get_by_name(domain)
         self.fields['role'].choices = role_choices
-        if domain_obj.commtrack_enabled:
+        if domain_obj:
```

Comment:
Isn't locations privilege-restricted?

It also seems odd to me that this would *always* be visible, as I don't think all projects use locations. Maybe it's worth checking to see if the project has any locations defined?

Reply:
agreed [b83869f](https://github.com/dimagi/commcare-hq/pull/34466/commits/b83869f47a02da39ac65cb7c33913ff107e2f1ff)